frontend/src/component/common/Table/cells/FeatureSeenCell/LastSeenTooltip.tsx

```diff
@@ -0,0 +1,137 @@
+import { styled, SxProps, Theme, useTheme } from '@mui/material';
+import TimeAgo from 'react-timeago';
+import { IEnvironments } from 'interfaces/featureToggle';
+import React from 'react';
+import { ConditionallyRender } from 'component/common/ConditionallyRender/ConditionallyRender';
+
+const StyledDescription = styled(
+    'div',
+    {}
+)(({ theme }) => ({
+    width: '100%',
+    maxWidth: theme.spacing(50),
+    padding: theme.spacing(1),
+    backgroundColor: theme.palette.background.paper,
+    color: theme.palette.text.secondary,
+    fontSize: theme.fontSizes.smallBody,
+    borderRadius: theme.shape.borderRadiusMedium,
+}));
+
+const StyledDescriptionBlock = styled('div')({
+    display: 'flex',
+    flexDirection: 'row',
+});
+
+const StyledDescriptionHeader = styled('p')(({ theme }) => ({
+    color: theme.palette.text.primary,
+    fontSize: theme.fontSizes.smallBody,
+    fontWeight: theme.fontWeight.bold,
+    marginBottom: theme.spacing(1),
+}));
+
+const StyledDescriptionBlockHeader = styled('p')(({ theme }) => ({
+    color: theme.palette.text.primary,
+    fontSize: theme.fontSizes.smallBody,
+    fontWeight: theme.fontWeight.bold,
+    marginBottom: theme.spacing(1),
+    width: '50%',
+}));
+
+const StyledDescriptionSubHeader = styled('p')(({ theme }) => ({
+    fontSize: theme.fontSizes.smallBody,
+    margin: theme.spacing(2, 0),
+}));
+
+interface ILastSeenTooltipProps {
+    environments?: IEnvironments[];
+    className?: string;
+    sx?: SxProps<Theme>;
+}
+
+const useLastSeeneColor = () => {
+    const theme = useTheme();
+
+    return (unit?: string): string => {
```

Comment:
Duplicate with `useFeatureColor`?

Reply:
sort of - gives different colors for the text

Reply:
Refactored out the hook